CHANGELOG.md

```diff
@@ -6,7 +6,8 @@
     - General
     - `icu_calendar`
       - Fix `und-SA-u-ca-islamic` (unicode-org#6736)
-    - 
+    - `icu_properties`
+      - Add support for missing binary properties.
```

Comment:
does this add support for all missing binary properties, or only some that you need? if it's not all, please update this